go.mod

```diff
@@ -110,7 +110,7 @@ require (
 	github.com/moby/spdystream v0.2.0 // indirect
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
-	github.com/opencontainers/runtime-tools v0.0.0-20190417131837-cd1349b7c47e // indirect
+	github.com/opencontainers/runtime-tools v0.9.0
```

Comment:
This one should now be moved to the other block (as it's no longer an `// indirect`. While doing so, could you also add a comment after it, indicating that the actual version is replaced?

Reply:
Moved to the other block and added a comment about replace.